### DISCUSSION TITLE
Custom Client Generator not Latest version

### DISCUSSION THREAD

**Discussion by xBr0th3rx** [rustdesk/rustdesk-server-pro#463] [General]
If I rebuild in the Custom Client Generator then only 1.3.2 is downloaded and not the latest 1.3.3.

**Comment by rustdesk** [comment]
New version does not always mean better, e.g. issues: https://github.com/rustdesk/rustdesk/discussions/10102, https://github.com/rustdesk/rustdesk/issues/10084
We will choose our time to upgrade custom client to new version when I think it is the best time. Do not hurry.

# 1.3.3 introduced big break change, there might be serious compatibility issue, we chose to be conservative, let it test more by free users.

We will publish an announcement in  https://github.com/rustdesk/rustdesk-server-pro/discussions and pin it every time when custom client version is updated.

**Comment by rustdesk** [comment]
We will publish an announcement in  https://github.com/rustdesk/rustdesk-server-pro/discussions and pin it every time when custom client version is updated.
